rust/parser/typeql.pest

```diff
@@ -588,31 +588,14 @@ ARROW = _{ "->" }
 
 // FRAGMENTS OF KEYWORDS =======================================================
 
-IDENTIFIER_CHAR = @{ ASCII_ALPHA
-                   | '\u{00C0}'..'\u{00D6}'
-                   | '\u{00D8}'..'\u{00F6}'
-                   | '\u{00F8}'..'\u{02FF}'
-                   | '\u{0370}'..'\u{037D}'
-                   | '\u{037F}'..'\u{1FFF}'
-                   | '\u{200C}'..'\u{200D}'
-                   | '\u{2070}'..'\u{218F}'
-                   | '\u{2C00}'..'\u{2FEF}'
-                   | '\u{3001}'..'\u{D7FF}'
-                   | '\u{F900}'..'\u{FDCF}'
-                   | '\u{FDF0}'..'\u{FFFD}'
-                   }
-IDENTIFIER_CONNECTOR = @{ "_"
-                        | "-"
-                        | "\u{00B7}"
-                        | '\u{0300}'..'\u{036F}'
-                        | '\u{203F}'..'\u{2040}'
-                        }
+IDENTIFIER_START = @{ XID_START }
+IDENTIFIER_CONTINUE = @{ "-" | XID_CONTINUE }
```

Comment:
`-` is not in XID_CONTINUE, but a valid continuation character in TypeQL identifiers.